core-bundle/src/Resources/contao/modules/ModuleBooknav.php

```diff
@@ -201,13 +201,6 @@ protected function getBookPages($intParentId, $groups, $time)
 		{
 			$objPage->loadDetails();
 
-			// Hide the page if it is not protected and only visible to guests (backwards compatibility)
-			if ($objPage->guests && !$objPage->protected && $isMember)
```

Comment:
this should be a separate PR (including `tl_page.guests` and migration)

Reply:
Reverted in 965ab6aa605345f3e860857032cebdeb8a3bdd8f.